### ISSUE TITLE
Hello author, how should I use my own data to make training set and test set?

### ISSUE THREAD

**Issue by Pan-long-ming** [XinJCheng/CSPN#53]
Hello author, how should I use my own data to make training set and test set?
